datahub-project/datahub
# API test for managing secrets privilege

API test for managing secrets privilege

Test steps:
1. Disable 'All users' privileges
2. Create a new user  
3. Verify new user can't create secrets
4. Assign privileges to the new user to manage secrets
5. Verify new user can create and manage secrets
6. Remove the policy
7. Ensure the user can't create a secret after the policy is removed
8. Remove test user
9. Restore All users privileges


## Checklist
